CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# v17.11.1 (Wed Aug 17 2022)
```

Comment:
Don't add CHANGLOG manually.

Reply:
Followed CONTRIBUTE.md
![image](https://user-images.githubusercontent.com/2663460/185126003-9dabc44f-4ba0-45e3-8c80-251b458c5622.png)
But maybe misinterpreted...

Reply:
CONTRIBUTE.md seems to be outdated :D

Reply:
Shall I remove my entry in CHANGELOG completely or just the first row?

Reply:
Remove your changes